examples/product-sample-supabase-kt/app/build.gradle

```diff
@@ -0,0 +1,98 @@
+}
+
+android {
+    namespace 'com.example.manageproducts'
+    compileSdk 33
+
+    defaultConfig {
+        applicationId "com.example.manageproducts"
+        minSdk 28
+        targetSdk 33
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+        vectorDrawables {
+            useSupportLibrary true
+        }
+        Properties properties = new Properties()
+        properties.load(project.rootProject.file("local.properties").newDataInputStream())
+        buildConfigField("String", "API_KEY", "\"${properties.getProperty("API_KEY")}\"")
+        buildConfigField("String", "SECRET", "\"${properties.getProperty("SECRET")}\"")
+        buildConfigField("String", "SUPABASE_URL", "\"${properties.getProperty("SUPABASE_URL")}\"")
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+    kotlinOptions {
+        jvmTarget = '1.8'
+    }
+    buildFeatures {
+        compose true
+    }
+    composeOptions {
+        kotlinCompilerExtensionVersion '1.2.0'
+    }
+    packagingOptions {
+        resources {
+            excludes += '/META-INF/{AL2.0,LGPL2.1}'
+        }
+    }
+}
+
+dependencies {
+
+    implementation 'androidx.core:core-ktx:1.7.0'
+    implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.3.1'
+    implementation 'androidx.activity:activity-compose:1.3.1'
+    implementation "androidx.compose.ui:ui:$compose_version"
+    implementation "androidx.compose.ui:ui-tooling-preview:$compose_version"
+    implementation 'androidx.compose.material3:material3:1.0.0-alpha11'
+    testImplementation 'junit:junit:4.13.2'
+    androidTestImplementation 'androidx.test.ext:junit:1.1.5'
+    androidTestImplementation 'androidx.test.espresso:espresso-core:3.5.1'
+    androidTestImplementation "androidx.compose.ui:ui-test-junit4:$compose_version"
+    debugImplementation "androidx.compose.ui:ui-tooling:$compose_version"
+    debugImplementation "androidx.compose.ui:ui-test-manifest:$compose_version"
+
+    implementation 'androidx.compose.material3:material3:1.0.0-alpha02'
+    implementation "androidx.compose.material:material:1.1.1"
+    
+    // Supabase setup
+    implementation("io.ktor:ktor-client-cio:1.6.0")
+    implementation("io.github.jan-tennert.supabase:postgrest-kt-android:1.0.0")
+    implementation "io.ktor:ktor-client-android:2.2.1"
+    implementation "io.ktor:ktor-utils-jvm:1.6.0"
+    implementation "io.ktor:ktor-client-logging-jvm:1.6.0"
+    implementation("io.ktor:ktor-client-core:2.2.4")
+    implementation("io.github.jan-tennert.supabase:storage-kt:1.0.0")
+    implementation "io.coil-kt:coil-compose:1.3.2"
```

Comment:
Is there a reason why ktor is so outdated and also have different versions? Same with the Kotlin version `1.6.21`, is there a reason why this is so old?

Reply:
This is because of me using the old. Things are fine with latest versions, updated